src/app/app.component.ts

```diff
@@ -402,6 +403,54 @@ export class AppComponent implements OnInit {
     this.collections   = Collections.getCollections();
     this.series        = SeriesVolumes.getSeries();
     this.seriesVolumes = SeriesVolumes.getSeriesVolumes();
+    this.comicsInReadingOrder = _.cloneDeep(this.comics);
+
+    let positionIterator = 0;
+    _.each(this.comicsInReadingOrder, (comic) => {
+      let currentSeriesVolume = this.seriesVolumes[_.findKey(this.seriesVolumes, { id: comic.seriesVolumeId })];
+      if (!currentSeriesVolume) {
+        throw new Error(comic.seriesVolumeId + ' not found');
+      }
+
+      positionIterator++;
+      // initialise
+      comic.containerStyles = { 'left.px': null, 'top.px': null, 'width.px': null };
+      comic.classes = { stickyBottom: false, stickyLeft: false, stickyRight: false, stickyTop: false };
+      comic.styles = { background: null, color: null, 'marginLeft.px': null, 'marginTop.px': null};
+
+      // Horizontal positioning
+      // Klaus This is creating a problem with double spacing and I can't figure out what to do
```

Comment:
I can't load the page to see what you mean, it just crashes the tab. Can you explain what double spacing means?